HCP docs update for IBM Power (#92359)
openshift/openshift-docs: modules/hcp-create-heterogeneous-nodepools.adoc

// Module included in the following assemblies:
//
// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-create-heterogeneous-nodepools_{context}"]
= Creating heterogeneous node pools

You can create heterogeneous node pools by using the `NodePool` custom resource (CR).

.Procedure

> * To define a `NodePool` CR, create a YAML file similar to the following example:

Reviewer: Previous 1-step procedures that created a CR used the `<<"EOF" | oc apply -f -` command. This one just creates the yaml. I'm sure it's fine, just calling it out.

Author: yes, I this it looks good as is.

// Module included in the following assemblies:
//
// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-create-heterogeneous-nodepools_{context}"]
= Creating heterogeneous node pools

You can create heterogeneous node pools by using the `NodePool` custom resource (CR).

.Procedure

> * To create heterogeneous node pools, create a YAML file that defines the `NodePool` CR:

Reviewer 2: ```suggestion
* To define a `NodePool` CR, create a YAML file similar to the following example:
```

+
[source,yaml]
----
apiVersion: hypershift.openshift.io/v1beta1
kind: NodePool
metadata:
  name: <hosted_cluster_name>
  namespace: <clusters_namespace>
spec:
  arch: <arch_ppc64le>
  clusterName: <hosted_cluster_name>
  management:
    autoRepair: false
    upgradeType: InPlace
  nodeDrainTimeout: 0s
  nodeVolumeDetachTimeout: 0s
  platform:
    agent:
      agentLabelSelector:
        matchLabels:

>           inventory.agent-install.openshift.io/cpu-architecture: <arch_ppc64le>
>     type: Agent
>   release:
>     image: quay.io/openshift-release-dev/ocp-release:<ocp_release>
>   replicas: 0
> ----

Reviewer 2: ```suggestion
          inventory.agent-install.openshift.io/cpu-architecture: <arch_ppc64le> <1>
    type: Agent
  release:
    image: quay.io/openshift-release-dev/ocp-release:<ocp_release>
  replicas: 0
----
<1> This selector block is selects the agents that match the specified label. 
To create a node pool of architecture `ppc64le` with zero replicas, specify `ppc64le`.
This ensures that it selects only agents from `ppc64le` architecture when it scales.
```

> +
> . To create a node pool of architecture `ppc64le` with zero replicas, see the following configuration:
> +
> [source,yaml]
> ----
> # ...
> agentLabelSelector:
>     matchLabels:
>         inventory.agent-install.openshift.io/cpu-architecture: ppc64le
> # ...
> ----
> +
> This selector block is used to select the agents which are matching the given label. Which will ensure it will select only the agents from `ppc64le` architecture when it is scaled.

Reviewer 2: This would make more sense as a defined valid value for the parameter in the YAML sample
```suggestion
```
